src/main/java/duke/Event.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDate;
+
+public class Event extends Task{
+    private static String taskType = "EVENT";
+//    private String startTime;
```

Comment:
Should we remove these comments since it's not used as an attribute of the class or for documentation purposes?